### PR TITLE
Mention invalid package name in error message

### PR DESCRIPTION
**Pull request by timhoffm**
### Description

This helps finding the source of the error.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](https://github.com/conda/conda/blob/master/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by timhoffm** [comment]
I have sent a signed CLA - this should be in process.

**Comment by timhoffm** [comment]
@conda-bot check

**Review by jezdez — APPROVED** [review]

**Comment by jezdez** [comment]
Oh wait, this needs a news item :)

**Comment by timhoffm** [comment]
Whoops, sorry for the mess.

File: news/11601-mention-invalid-name-in-error-message [hunk 1]
```diff
@@ -0,0 +1,21 @@
+### Enhancements
+
+* If an invalid package name is found in a specification, mention that name
+  in the error message.
+
```
Comment (kenodegard) [inline news/11601-mention-invalid-name-in-error-message:5]:
```suggestion
* Include the invalid package name in the error message. (#11601)
```

**Review by kenodegard — APPROVED** [review]

**Pull request merged**